packages/accordions/src/styled/StyledLabel.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import {
+  getColor,
+  getLineHeight,
+  retrieveComponentStyles,
+  DEFAULT_THEME
+} from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'steppers.step_label';
+
+interface IStyledLabelProps {
+  isActive?: boolean;
+  isHorizontal?: boolean;
+}
+
+export const StyledLabel = styled.div.attrs<IStyledLabelProps>({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledLabelProps>`
+  display: ${props => !props.isHorizontal && 'flex'};
+  flex: ${props => props.isHorizontal && '1'};
+  flex-direction: ${props => (props.isHorizontal ? 'column' : 'row')};
+  align-items: center;
+  text-align: center;
```

Comment:
Might need some logic here to left align Label in horizontal applications for when it wraps: 
![image](https://user-images.githubusercontent.com/29072694/76112709-99028100-5f97-11ea-81ba-a87dd1098014.png)

Reply:
Spoke to @rossmoody, leaving as is because only browser dev tools can force a width that results in this UI.

Reply:
I think the alignment issue has been addressed:

<img width="269" alt="Screen Shot 2020-03-09 at 3 28 49 PM" src="https://user-images.githubusercontent.com/1811365/76262854-b72be380-621a-11ea-8d8b-7d44c4f94763.png">

I'll resolve this thread and move all other sizing issues regarding `Stepper` being rendered in small containers to [this thread](https://github.com/zendeskgarden/react-components/pull/658#discussion_r389900897).